uaclient/event_logger.py

```diff
@@ -41,6 +42,29 @@ class EventLoggerMode(enum.Enum):
     YAML = object()
 
 
+def format_machine_readable_output(status: Dict[str, Any]) -> Dict[str, Any]:
+    status["environment_vars"] = [
+        {"name": name, "value": value}
+        for name, value in sorted(os.environ.items())
+        if name.lower() in CONFIG_FIELD_ENVVAR_ALLOWLIST
+        or name.startswith("UA_FEATURES")
+        or name == "UA_CONFIG_FILE"
+    ]
+
+    if not status.get("simulated"):
+        available_services = [
+            service
+            for service in status.get("services", [])
+            if service.get("available", "yes") == "yes"
+        ]
+        status["services"] = available_services
+
+    # We don't need the origin info in the json output
+    status.pop("origin", "")
+
+    return status
```

Comment:
I expected this function to be in the `status` module. IMO it's not so much "formatting" as it is adding extra status information. What do you think?

Reply:
We could do that, but right now this will generate a circular dependency between `status` and `event_logger`. Which we could solve by adding the import directly on the event_logger function that requires the function.
Since we will probably do a major refactoring of how the event_logger work in the near future, I would not change this right now, but let me know what you think

Reply:
I see.

I guess I think this function should be called at the end of `uaclient.status.status` and `uaclient.status.simulate_status` instead of in the event_logger.

But I can see how that refactor could be considered out of scope, so not needed to merge this

Reply:
Agreed